collectors/freeipmi.plugin/metadata.yaml

```diff
@@ -42,36 +42,204 @@ modules:
+              required: false
+            - name: assume-bmc-owner
+              description: assume the BMC is the sensor owner no matter what (usually bridging is required too)
+              default_value: disabled
+              required: false
+            - name: hostname HOST
+              description: Connect to remote IPMI host.
+              default_value: local
+              required: false
+            - name: username USER
+              description: Connect to remote IPMI host.
+              default_value: IPMI
+              required: false
+            - name: password PASS
+              description: Connect to remote IPMI host.
+              default_value: processor
+              required: false
+            - name: noauthcodecheck / no-auth-code-check
+              description: don't check the authentication codes returned
+              default_value: ""
+              required: false
+            - name: driver-type IPMIDRIVER
+              description: Specify the driver type to use instead of doing an auto selection. The currently available outofband drivers are LAN and LAN_2_0, which  perform  IPMI  1.5  and  IPMI  2.0 respectively. The currently available inband drivers are KCS, SSIF, OPENIPMI and SUNBMC.
+              default_value: ""
+              required: false
+            - name: sdr-cache-dir PATH
+              description: directory for SDR cache files
+              default_value: /tmp
+              required: false
+            - name: sensor-config-file FILE
+              description: filename to read sensor configuration
+              default_value: system default
+              required: false
+            - name: sel-config-file FILE
+              description: filename to read sel configuration
+              default_value: system default
+              required: false
+            - name: ignore N1,N2,N3,...
+              description: sensor IDs to ignore
+              default_value: ""
+              required: false
+            - name: ignore-status N1,N2,N3,...
+              description: sensor IDs to ignore status (nominal/warning/critical)
+              default_value: ""
+              required: false
+            - name: -v / -V / version
+              description: print version and exit
+              default_value: ""
               required: false
         examples:
           folding:
             enabled: true
-            title: ""
-          list: []
+            title: "Config"
+          list:
+            - name: 5 second update rate and SEL collection
+              description: Append to `command options =` the settings you need. The minimum `update every` is 5 (enforced internally by the plugin). IPMI is slow and CPU hungry. So, once every 5 seconds is pretty acceptable.
+              config: |
+                [plugin:freeipmi]
+                  update every = 5
+                  command options = sel
+              folding:
+                enabled: false
```

Comment:
sel is enabled by default, also don't combine changing data collection freq with something else. If you want this should be 2 different examples:

 - disable SEL
 - change (increase) data collection frequency

Reply:
oh okay, will make two then, yeah